### PR TITLE
#6916 - Added tooltips to the numerous elements listed in the issue.

### PR DESCRIPTION
**Pull request by alexander-fedorenko**
## Description
Added tooltips to the numerous elements listed in the issue.

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

#6916

**What is the new behavior?**
Elements will have translatable tooltips.

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

**Review by MV88 — CHANGES_REQUESTED** [review]
hi @alexander-fedorenko 

please have a look at my comments

for Future PRs ignore locales outside than the 5 supported ones and 
focus translations for these ones
- it
- en
- fr
- de
- es

File: web/client/plugins/manager/ManagerMenu.jsx [hunk 1]
```diff
@@ -11,14 +11,19 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import assign from 'object-assign';
 import { isPageConfigured } from '../../selectors/plugins';
+import tooltip from '../../components/misc/enhancers/tooltip';
+
 import { DropdownButton, Glyphicon, MenuItem } from 'react-bootstrap';
 
+const DropdownButtonT = tooltip(DropdownButton);
```
[the remark this answers is not in the document]
Comment (alexander-fedorenko) [reply web/client/plugins/manager/ManagerMenu.jsx]: Thanks, renamed it

File: web/client/test-resources/data.es-ES.json [hunk 2]
```diff
@@ -248,7 +248,7 @@
             },
             "layoutWarning": "Layout no permitido"
         },
-        "backgroundSwither":{
+        "backgroundSwitcher":{
```
[the remark this answers is not in the document]
Comment (alexander-fedorenko) [reply web/client/test-resources/data.es-ES.json]: Changes are reverted

**Review by MV88 — APPROVED** [review]

**Comment by MV88** [comment]
@ElenaGallo please test in DEV

**Pull request merged**